### ISSUE TITLE
[Errata] Chapter 2 Question 1: option C seems to be correct

### ISSUE THREAD

**Issue by guiNovais** [awsccp/awsccp.github.io#55]
**Please describe the error you're reporting**
It seems to me the option **C** is correct.

The answer to this question says: "any combination of t2.micro instances can be run up to a total of 750 hours per month". The option **C** says "a single t2.micro instance type instance for 750 hours per month", which seems to fits the first description.

**Please indicate the chapter or appendix, section, and page number where the error appears (if applicable)**
Chapter 2, Question 1 (pg 28)
Appendix A, Chapter 2, Answer 1 (pg 232)

**Please include any other comments or questions**
It does not turn the answer D incorrect.

**Comment by dbclinton** [comment]
Yup. That's a problem. Technically, both C and D are correct. I'll fix this for the new edition (which is coming out in a couple of  months).
Thanks!
